create_bags/digitization_pipeline.py

```diff
@@ -37,6 +37,9 @@ def run(self, rights_ids):
                 logging.info(
                     "Bag successfully created: {}".format(created_bag))
                 remove_copied_files(list_of_files)
+                if Path(self.tmp_dir, refid, "service").is_dir():
+                    Path(self.tmp_dir, refid, "service").rmdir()
+                Path(self.tmp_dir, refid).rmdir()
```

Comment:
I would lobby for changing the name of this method to something more like `remove_bag` and moving lines 40-42 to that method. Might also make the case for just using `shutil.rmtree` in this instance.